glide.lock

```diff
@@ -17,14 +19,26 @@ imports:
   - internal/sysinfo
   - internal/utilization
 - name: github.com/sirupsen/logrus
-  version: ba1b36c82c5e05c4f912a88eab0dcd91a171688f
+  version: dae0fa8d5b0c810a8ab733fbd5510c7cae84eca4
+- name: golang.org/x/crypto
+  version: a5d413f7728c81fb97d96a2b722368945f651e78
+  subpackages:
+  - ssh/terminal
 - name: golang.org/x/sys
-  version: 7a56174f0086b32866ebd746a794417edbc678a1
```

Comment:
You picked up updates for many other non-versioned dependencies. If you want to be rigorous for FDA purposes you should revert all the changes in glide.lock except logrus and merry.

Alternatively pin all the versions in glide.yaml.